example/spec/startSF/startSfSpec.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const { lambda, sfn, sqs } = require('@cumulus/common/aws');
+const { loadConfig } = require('../helpers/testUtils');
+
+const config = loadConfig();
+
+const testName = `${config.stackName}-testStartSf`;
+
+const passSfName = `${config.stackName}-passTestSf`;
+const passSfDef = {
+  Comment: 'Pass-only step function',
+  StartAt: 'PassState',
+  States: {
+    PassState: {
+      Type: 'Pass',
+      Result: '$.payload',
+      End: true
+    }
+  }
+};
+const passSfRoleArn = `arn:aws:iam::${config.awsAccountId}:role/${config.stackName}-steprole`;
+
+const passSfParams = {
+  name: passSfName,
+  definition: JSON.stringify(passSfDef),
+  roleArn: passSfRoleArn
+};
+
+const sfStarterName = `${config.stackName}-sqs2sf`;
+
+function generateStartSfMessages(num, workflowArn) {
+  const arr = [];
+  for (let i = 0; i < num; i += 1) {
+    arr.push({ cumulus_meta: { state_machine: workflowArn }, payload: `message #${i}` });
+  }
+  return arr;
+}
+
+describe('the sf-starter lambda function', () => {
+  let queueUrl;
+
+  beforeAll(async () => {
+    const { QueueUrl } = await sqs().createQueue({
+      QueueName: `${testName}Queue`
```

Comment:
Can you suffix a random few characters there so that we can run IT more than once per minute?
`AWS.SimpleQueueService.QueueDeletedRecently: You must wait 60 seconds after deleting a queue before you can create another with the same name.`